tests/spread/core22/manifest/manifest-info-cmdline/task.yaml

```diff
@@ -0,0 +1,28 @@
+summary: Test manifest file creation
+
+prepare: |
+  snap install review-tools
+
+restore: |
+  snapcraft clean
+  rm -f ./*.snap
+
+execute: |
+  snapcraft --use-lxd --enable-manifest --manifest-image-information='{"test-var": "value"}'
+
+  unsquashfs manifest_0.1_*.snap
+  grep '^  test-var: value' squashfs-root/snap/manifest.yaml
+  rm -Rf squashfs-root
+
+  cp manifest_0.1_*.snap ~
+  review-tools.snap-review ~/manifest_0.1_*.snap
+  rm ~/manifest_0.1_*.snap
+
+  # also test in destructive mode
+  snapcraft --destructive-mode --enable-manifest --manifest-image-information='{"test-var": "value"}'
+
+  grep '^  test-var: value' prime/snap/manifest.yaml
+
+  cp manifest_0.1_*.snap ~
+  review-tools.snap-review ~/manifest_0.1_*.snap
+  rm ~/manifest_0.1_*.snap
```

Comment:
this should be done on restore in case review-tools.snap-review fails

Reply:
rm -f

Reply:
Changed to remove file from home in all 3 tests.